assets/html/custombar.html

```diff
@@ -7,18 +21,33 @@
       <div class="bounce3"></div>
     </div>
   </div>
+  <label class="letter-label">
+        <input class="js-bag-input letters-input" type="text" maxLength="2" pattern="[A-Z]" placeholder="A-Z"/>
+  </label>
+  <div class="tab-cnr">
+    <div class="tab-buffer">
```

Comment:
yeah, instead of doing `border-bottom: none;` for the active tab (which will also change the height by 1 pixel) you can do `border-bottom-color: transparent;` or `border-bottom-color: white;`